src/main/resources/init_db.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA `taxi_service` DEFAULT CHARACTER SET utf8 ;
+
+CREATE TABLE `taxi_service` (
```

Comment:
schema is called taxi_service
all the tables inside this schema should have informative names
```suggestion
CREATE SCHEMA `taxi_service` DEFAULT CHARACTER SET utf8 ;

CREATE TABLE `taxi_service`.`manufacturers` (
```